auth/src/swig/auth.i

```diff
@@ -815,12 +831,17 @@ static CppInstanceManager<Auth> g_auth_instances;
   ///
   /// An error is returned, if the token is invalid, expired or otherwise not
   /// accepted by the server.
-  public System.Threading.Tasks.Task<FirebaseUser> SignInWithCredentialAsync(
+  ///
+  /// @deprecated This function is deprecated in favor of `Task<AuthResult>`
```

Comment:
I would update these to "This method is deprecated in favor of methods that return `Task<AuthResult>`. Please use..."